modules/update-service-install-web-console.adoc

```diff
@@ -0,0 +1,34 @@
+[id="update-service-install-web-console_{context}"]
+= Installing the {product-title} Update Service by using the web console
+
+You can use the {product-title} web console to install the {product-title} Update Service Operator.
+
+.Procedure
+
+To install the {product-title} Update Service Operator using the {product-title} web console:
+
+. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+. Choose *{product-title} Update Service* from the list of available Operators, and click *Install*.
+
+. FIXME: Does this apply? Ensure that the *All namespaces on the cluster* is selected under *Installation Mode*.
```

Comment:
It should be defaulted to `A specific namespace on the cluster` which appears to be something we can restrict it to as well.

Reply:
@wking, did this default behavior change?

Reply:
I'm agnostic about which namespaces the operator gets installed to, or if it watches all namespaces.  @jottofar is more familiar with what our bundle recommends.  The text we're commenting on I probably cribbed from the logging docs, which may set things up differently.

Reply:
It should come up defaulted to single namespace of "openshift-update-service". May be safer to have user leave it defaulted as did not do a lot of testing in the end with any other namespace although can't think of a reason why it would be an issue.

Reply:
I was incorrect. It comes up defaulted to `All namespaces on the cluster`. We should instruct user to select `A specific namespace on the cluster` and that choice will have the `Operator recommended Namespace` of `openshift-updateservice` which should be used. Screenshot:

![Screenshot from 2021-02-08 15-56-03](https://user-images.githubusercontent.com/28737257/107280843-db91d080-6a26-11eb-93a6-85e02e83a4d5.png)